File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/SmallbotProduction.java
package org.firstinspires.ftc.teamcode.opmodes;

import org.firstinspires.ftc.teamcode.features.PowerplayFourMotorArm;
import org.firstinspires.ftc.teamcode.features.Hand;
import org.firstinspires.ftc.teamcode.features.PowerplayMecanumDrivetrain;
import org.firstinspires.ftc.teamcode.internals.registration.OperationMode;
import org.firstinspires.ftc.teamcode.internals.registration.TeleOperation;

public class SmallbotProduction extends OperationMode implements TeleOperation {

    @Override
    public void construct() {
        registerFeature(new PowerplayMecanumDrivetrain(false, true));
        registerFeature(new PowerplayFourMotorArm());
        registerFeature(new Hand());
    }

    @Override
    public void run() {}

}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/SmallbotProduction.kt
package org.firstinspires.ftc.teamcode.opmodes

import org.firstinspires.ftc.teamcode.features.Hand
import org.firstinspires.ftc.teamcode.features.PowerplayFourMotorArm
import org.firstinspires.ftc.teamcode.features.PowerplayMecanumDrivetrain
import org.firstinspires.ftc.teamcode.internals.registration.OperationMode
import org.firstinspires.ftc.teamcode.internals.registration.TeleOperation

class SmallbotProduction : OperationMode(), TeleOperation {
    override fun construct() {
        registerFeature(PowerplayMecanumDrivetrain(false, true))
        registerFeature(PowerplayFourMotorArm())
        registerFeature(Hand())
    }

    override fun run() {}
}
